src/rnp/fficli.h

```diff
@@ -54,6 +54,9 @@ class cli_rnp_t {
     FILE *    userio_in{};  /* file pointer for user's inputs */
     FILE *    userio_out{}; /* file pointer for user's outputs */
     int       pswdtries{};  /* number of password tries, -1 for unlimited */
+    bool      reuse_password_for_subkey{};
+    char *    reuse_primary_fprint{};
+    char *    reused_password{};
```

Comment:
@ni4 Can I use rnp::secure_vector here for "reused_password"? I think std::string is not suitable for security reasons.

Reply:
@antonsviridenko Generally in CLI we should not link to/use code from the layers below the FFI interface. Also you may use rnp_buffer_clear() from the FFI interface once cleaning up.

Reply:
@ni4 Do you mean to keep `reused_pasword` as is, as a char * pointer? I thought about calling `rnp_buffer_clear()` on std::string::c_str(), but

> Writing to the character array accessed through c_str() is undefined behavior.